src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let temporarySum = 0;
+
+  return function add(input) {
+    if (typeof input === 'undefined') {
```

Comment:
would be easier to do it like:
```suggestion
    if (input === undefined) {
```